### PR TITLE
Updated pot file and added german translations 

### PR DESCRIPTION
**Pull request by JonathanSteinbuch**
I saw that the diff files are pretty large, presumably because the pot file was not updated in a while. So if there is a different process for such pot changes, let me know and ignore this request.

I only added five lines of german translation for the peaceful mode and geologist sign durability addons of mine that were commited recently.

### CODE REVIEW

**Comment by Flamefire** [comment]
The large diff is usually caused by missed sorting. If you are on linux with a generated build tree you can run `make -B translations` which will automatically sort the files

**Comment by JonathanSteinbuch** [comment]
Thanks, @Flamefire. Indeed I had thought I did what you said, but I did it wrong. 

The sorting seems to be correct now. There are still quite some changes, but now it's mostly comments that say that line numbers changed and a few new texts and some apparently deleted ones and no reorderings. Those lines were also automatically changed in the other languages .po-files but I didn't add translations to those, so I didn't add those files to the commit

File: rttr-de.po [hunk 1]
```diff
@@ -4170,5 +4242,19 @@ msgstr "Ödland"
...
+#, fuzzy
+#~ msgid "Starting sound conversion..."
+#~ msgstr "Starte Sound-Converter...\n"
+
+#, fuzzy
+#~ msgid "failed\n"
+#~ msgstr "fehlgeschlagen: %1%\n"
+
+#~ msgid "n/a"
+#~ msgstr "n/v"
```
Comment (Flamefire) [inline rttr-de.po]: Those can be removed (simply click accept):
```suggestion
```

**Review by Flamefire — APPROVED** [review]

**Comment by Flamefire** [comment]
Thanks a lot!

**Pull request merged**